### ISSUE TITLE
Wildly different example outputs in `sequence_classification`

### ISSUE THREAD

**Issue by Rattko** [ufal/npfl114#171]
Hi,

outputs in the 'Examples' section of the task `sequence_classification` are wildly different to what I'm getting locally. The outputs in the 'Tests' section are identical to mine and my solution also passed the tests in ReCodEx. Perhaps you used a different seed? I doubt there's such a huge difference between macOS and Linux.

Btw. it seems you swapped the order of 'Examples' and 'Tests' on the webpage :D

Example of the output I'm getting

```
$ python3 sequence_classification.py --rnn=LSTM --epochs=5 --hidden_layer=50

Epoch 1/5 loss: 0.6828 - accuracy: 0.5167 - val_loss: 0.6590 - val_accuracy: 0.5178
Epoch 2/5 loss: 0.6441 - accuracy: 0.5408 - val_loss: 0.6303 - val_accuracy: 0.5264
Epoch 3/5 loss: 0.6227 - accuracy: 0.5565 - val_loss: 0.6145 - val_accuracy: 0.5573
Epoch 4/5 loss: 0.6108 - accuracy: 0.5579 - val_loss: 0.6020 - val_accuracy: 0.5617
Epoch 5/5 loss: 0.5932 - accuracy: 0.5699 - val_loss: 0.5876 - val_accuracy: 0.5717
```

Cheers.

**Comment by foxik** [comment]
Hi,

what you are seeing is really just a difference between OSes/CPUs; when I run your ReCodEx solution on my laptop (and also on the ReCodEx machine), I get exactly the results from the Examples, i.e.,
```
$ python3 sequence_classification.py --rnn=LSTM --epochs=5 --hidden_layer=50
Epoch 1/5
625/625 [==============================] - 8s 10ms/step - loss: 0.6829 - accuracy: 0.5160 - val_loss: 0.6601 - val_accuracy: 0.5166
Epoch 2/5
625/625 [==============================] - 5s 8ms/step - loss: 0.6447 - accuracy: 0.5398 - val_loss: 0.6310 - val_accuracy: 0.5274
Epoch 3/5
625/625 [==============================] - 5s 8ms/step - loss: 0.6226 - accuracy: 0.5545 - val_loss: 0.6108 - val_accuracy: 0.5522
Epoch 4/5
625/625 [==============================] - 5s 8ms/step - loss: 0.5895 - accuracy: 0.5859 - val_loss: 0.5529 - val_accuracy: 0.6215
Epoch 5/5
625/625 [==============================] - 5s 8ms/step - loss: 0.4641 - accuracy: 0.7130 - val_loss: 0.3106 - val_accuracy: 0.8517
```

The gradients are exploding during the training (because the example with gradient clipping is training faster), so even a very small difference in loss computation can cascade into a large difference in accuracy.

Yes, I swapped the order, mostly because of ReCodEx -- you see all the tests/examples always expanded there, so I placed the tests first (and numbered them) to make it easier to find the inputs corresponding to ReCodEx tests.
